### PR TITLE
radio/radioList Components

### PR DESCRIPTION
**Pull request by briancao**
added radio/radioList component
added labels to checkbox/checkboxList
edited prettier config
fix bug w/ text input

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,4 +9,13 @@ Thumbs.db
 
 # App files
 node_modules
-*.map
\ No newline at end of file
+*.map
+\.vs/ProjectSettings\.json
```
Comment (mikecao) [inline .gitignore]: Just ignore the entire `.vs` and `.vscode` folders. And no need to escape the dot.

File: package.json [hunk 2]
```diff
@@ -14,10 +14,21 @@
     "start": "webpack-dev-server --hot --inline --history-api-fallback",
     "build": "webpack -p --config webpack.config.js",
     "lint": "eslint --ext .js,.jsx src || exit 0",
-    "test": "jest || exit 0"
+    "test": "jest --watch"
```
Comment (mikecao) [inline package.json]: Make a separate action for watch, `test:watch`, the default should not be to watch.

File: src/components/input/CheckboxList.js [hunk 3]
```diff
@@ -1,52 +1,53 @@
...
-        ));
-
         return (
             <div className={styles.checkboxList}>
-                {checkBoxes}
+                {items.map((item, i) => (
+                    <div key={i}>
+                        <Checkbox
+                            name={item.name}
+                            value={(values.includes(item.value))}
```
Comment (mikecao) [inline src/components/input/CheckboxList.js]: Are you using value as checked? That doesn't seem right, they are separate things.

Comment (briancao) [reply src/components/input/CheckboxList.js]: In CheckboxComponent, I'm passing value into checked && value
image

Then CheckboxList receives an array of items, which is all the selectable values, [{ name, value }]
And an array of values, which is all the selected values, [value]

File: .gitignore (hunk 1 above)
Comment (briancao) [reply .gitignore]: ignored.

File: package.json (hunk 2 above)
Comment (briancao) [reply package.json]: completed.

**Pull request merged**